apps/generator/test/integration.test.js

```diff
@@ -10,6 +10,7 @@ const dummySpecPath = path.resolve(__dirname, './docs/dummy.yml');
 const refSpecPath = path.resolve(__dirname, './docs/apiwithref.json');
 const refSpecFolder = path.resolve(__dirname, './docs/');
 const crypto = require('crypto');
+const fs = require('fs');
```

Comment:
please do not use `existsSync` and use `access` that is already required in line 6

Reply:
Thanks @derberg done